manage_user_page.php

```diff
@@ -358,24 +365,51 @@
 </div>
 </div>
 
-<div class="widget-toolbox padding-8 clearfix">
-	<div id="manage-user-edit-div" class="form-inline pull-left">
-		<form id="manage-user-edit-form" method="get" action="manage_user_edit_page.php" class="form-inline"
-			<?php # CSRF protection not required here - form does not result in modifications ?>>
-			<label class="inline" for="username"><?php echo lang_get( 'search' ) ?></label>
-			<input id="username" type="text" name="username" class="input-sm" value="" />
-			<input type="submit" class="btn btn-primary btn-sm btn-white btn-round" value="<?php echo lang_get( 'manage_user' ) ?>" />
-		</form>
-	</div>
-	<div class="btn-toolbar pull-right">
-		<?php
-		# @todo hack - pass in the hide inactive filter via cheating the actual filter value
-		print_page_links( 'manage_user_page.php', 1, $t_page_count, (int)$f_page_number, $f_filter . $t_hide_inactive_filter . $t_show_disabled_filter . "&amp;sort=$c_sort&amp;dir=$c_dir");
-		?>
-	</div>
-</div>
+<?php 
+draw_search_section('bottom',$t_total_user_count,$p_per_page,$t_page_count, $f_page_number, $t_filter_string); 
+?>
 </div>
 </div>
 </div>
 <?php
 layout_page_end();
+
+
+/**
+ * draws the search input and the page navigator
+ *
+ * @param string $p_where  just to create different HTML ID attributes.
+ * @param int    $p_user_count  how many users have been found.
+ * @param int    $p_per_page  how many records display per page.
+ * @param int    $p_page_count how many pages do we have.
+ * @param int    $p_page_number current page number.
+ * @param string $p_filter_string used by print_page_links() 
+ *
+ * @return void
+ */
+function draw_search_section($p_where,$p_user_count,$p_per_page,$p_page_count, $p_page_number, $p_filter_string) {
+
+	if( $p_user_count < $p_per_page ) {
+		return;
```

Comment:
Before this change, we were always able to search for a user by name.
Now there is use case where I am missing this functionality.

Assume there are 60 user names starting with character 'A'.
Click on `A` right beside the `All` button.
The search field is available in this case and you will be able to search for any user, not just the ones starting with 'A'.

Assume there are 40 user names starting with character 'A'.
Click on `A` right beside the `All` button.
The search field is not available in this case.